packages/mask/src/components/InjectedComponents/PluginSettingsDialog.tsx

```diff
@@ -82,19 +85,27 @@ export function PluginSettingsDialog() {
 
     useEffect(() => MaskMessages.events.ownPersonaChanged.on(retry), [retry])
 
+    const onOpenPopup = useCallback(Services.Helper.openPopupWindow, [])
```

Comment:
It's stable, no need to wrap with `useCallback`.

If you want an alias.
```suggestion
    const onOpenPopup = Services.Helper.openPopupWindow
```